### DISCUSSION TITLE
Transparency seems broken for the past hour or so

### DISCUSSION THREAD

**Discussion by Kibetho** [google/model-viewer#2386] [Q&A]
I've been publishing some 3D models to IPFS using model-viewer called externally to display them

![image](https://user-images.githubusercontent.com/37402533/118532863-56ae0580-b73f-11eb-9e7f-48f0d5e7f213.png)

Up until about an hour back the model was visible as in the image above, but if I go to it now via Mobile or desktop it is distorted, zoomed out and seems to contain an additional transparent artefact

![image](https://user-images.githubusercontent.com/37402533/118533000-77765b00-b73f-11eb-8ae8-2156151a5f7a.png)

With IPFS there should not have been any change to my file from when it was working until when it wasn't - so I'm wondering if something might have changed in a recent update.

Link to 3D model display - https://bafybeibcu6a7vj47jfpjnv763wue2vdkrkpylrggknmwkssj2uumjc77gm.ipfs.infura-ipfs.io/

**Comment by elalish** [comment]
Is the issue that you're getting that extra tall box below the normal model? It's possible this is due to our updating the three.js version. Still, you may have simply been lucky before; I'd check your glTF. In our [editor](https://modelviewer.dev/editor/) the validator says the file is fine but has a number of warnings that may be useful for debugging.

**Reply by Kibetho** [reply]
Thanks for the reply, yes that's the issue, can't replicate this in other web viewers or desktop software - this is a really simple model. I'll try to debug.

**Reply by Kibetho** [reply]
I can't seem to find an issue with this model

If I make a transparent tinted cube in blender

![image](https://user-images.githubusercontent.com/37402533/118544460-574d9880-b74d-11eb-8cb7-bfe8945ab758.png)

It creates another cube inside it with model-viewer
![image](https://user-images.githubusercontent.com/37402533/118544503-646a8780-b74d-11eb-97a1-46c7d7248fae.png)

**Reply by elalish** [reply]
Thanks, can you link that cube here? I bet you can work around the problem by making the transparent object not double-sided. We duplicate the geometry in this case, but it looks like a transform is getting lost. Your simple cube should help with debug.

**Reply by Kibetho** [reply]
Thanks for the swift reply I have uploaded the model to IPFS - https://ipfs.io/ipfs/QmQRBzTW8pivoGSiZY4ERoGQg24mBqgK6n1m2xtQpTrp6v?filename=boxtest556.glb

**Reply by elalish** [reply]
Found the problem: #2390.
